docs/docs/query-filters.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Query Filters with GraphQL in Gatsby
+---
+
+> This documentation is up to date as of `gatsby@2.23.0`.
+
+## Summary
+
+Gatsby stores all data loaded during the source-nodes phase in Redux and it allows you to write GraphQL queries to query that data. This data, stored as individual "nodes", can be searched through using a query language that is a subset of [MongoDb queries](https://docs.mongodb.com/manual/reference/operator/query/).
```

Comment:
I'd rather say that we are inspired by mongodb rather than claiming we implement a subset.